centreon/packages/ui/src/Snackbar/index.tsx

```diff
@@ -52,6 +59,9 @@ const Snackbar = React.forwardRef(
       closeSnackbar(id);
     };
 
+    const formatedMessage =
+      typeof message === 'string' ? <div>{sanitizedMessage}</div> : message;
+
```

Comment:
the stories show this component actually accepts `message: string | Array<string>`
when it's an array the formatting is not applied

do we have the use case of multiple messages in one snackbar ? cc/ @Thebarda 

if so, then we should support it and add the formatting there to (and fix this components props)

Reply:
For now, we have only used it for one message, but we can extend it to handle multiple messages. However, in order to achieve that, I will need to convert it from a hook to a simple function. Therefore, we can rename useSanitizedHTML to sanitizeHTML

Reply:
I've pushed new commits, you can check the new implementation

Reply:
the issue is that the props are not correct

it should be
```
export interface SnackbarProps {
  id: string | number;
  message: string | Record<string, string>; // check the story in storybook
  severity: Severity;
}
```

now, i'm not aware if this `Record<string, string>` is used anywhere else, or only in the story

but it should also be supported imo.